javascript/javascript2/week3/Warmup.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <style>
+        body {
+            height: 620px;
```

Comment:
What was the reason that you add here instead of the css file?

Reply:
Thank you for your feedback. Making less files.